cfgov/v1/management/commands/update_data_snapshot_values.py

```diff
@@ -0,0 +1,67 @@
+import ast
+
+from django.core.management.base import BaseCommand
+
+from v1.models.browse_page import BrowsePage
+from v1.tests.wagtail_pages.helpers import publish_changes
+
+
+class Command(BaseCommand):
+    help = 'Monthly updates to data snapshot values'
+
+    def add_arguments(self, parser):
+        """Adds all arguments to be processed."""
+        parser.add_argument(
+            '--markets',
+            nargs='?',
+            help='String representation of a list of markets'
+        )
+
+    def get_data_snapshots(self):
+        """ Gets all data snapshots from browse pages
+        Assumes there is one data snapshot per page
+        """
+        snapshots = []
+        for page in BrowsePage.objects.all():
```

Comment:
@chosak Thank *you* for the feedback! I talked with @hillaryj about this suggestion, and I think we feel that this would add a bit of (mental) overhead to people working with these pages (as in remembering if they should be adding a new `BrowsePage` or a new subpage -- though perhaps that's not really a concern since they have to be trained in it one way or the other) as well as pollute the codebase a little. I think it would make sense if the page differed in any way, but if the page type exists solely to speed up this lookup, it feels a bit like overengineering, particularly since the script only runs once a month and isn't slowing anything down. I think we'd like to proceed with this for now but if you feel strongly let's talk more.